core/executor/src/adapter/trie_db.rs

```diff
@@ -172,11 +170,13 @@ fn rand_remove_list<T: Clone>(keys: Vec<T>, num: usize) -> Vec<T> {
     for _ in 0..num {
         let tmp = rng.gen_range(0..len);
         let idx = idx_list.remove(tmp);
-        ret.push(keys[idx].to_owned());
+        ret.push(idx);
```

Comment:
Change `ret` to `HashSet`?

Reply:
This modification is based on the timing of the flush. If the amount of data exceeded when flushing is less than 100, the performance of vec is better than HashSet

Reply:
In most cases, the number of data that to be removed is greater than 100.